### PR TITLE
README.rst with new E2E architecture using DBus PhoneHome service

### PR DESCRIPTION
**Pull request by jframos**
#### Reviewres

@flopezag @pratid 
#### Description

Documentation of the new DBus and HTTP HomePhone services.

### CODE REVIEW

File: fiware-region-sanity-tests/README.rst [hunk 1]
```diff
@@ -23,6 +23,16 @@ Test environment
 __ `Python - Downloads`_
 __ `Vagrant - Downloads`_
 
+The E2E SNAT test uses DBus tech for communicating process. This one requires:
+
+- `Bbus`_ running and configured on your system
```
Comment (pratid) [inline fiware-region-sanity-tests/README.rst]: Typo: DBus

File: fiware-region-sanity-tests/README.rst [hunk 2]
```diff
@@ -225,6 +236,74 @@ provided as part of this component, which has to be installed together with the
...
 
 
+DBus and HTTP PhoneHome Service for E2E tests
+---------------------------------------------
+
+Some E2E test cases have been implemented to check the connection in both
+*Internet -> VM* and *VM -> Internet*.
+This test cases are:
+
+* Test whether it is possible to deploy and instance, assign an allocated
```
Comment (flopezag) [inline fiware-region-sanity-tests/README.rst]: "deploy and instance" or "deploy an instance"?

File: fiware-region-sanity-tests/README.rst [hunk 3]
```diff
@@ -225,6 +236,74 @@ provided as part of this component, which has to be installed together with the
...
+to the *HTTP PhoneHome service* running in the configured HOST:PORT
+(*phonehome_endpoint* configuration). If this value is not set, this test will be skipped.
+
+The test uses two components:
+
+- A HTTP/Dbus PhoneHome server, that is launched as a service in the same host where test is executed (with public IP).
+- A DBus client used by test implementation to wait for PhoneHome requests through the HTTP PhoneHome server.
+
+The implemented PhoneHome service uses the DBus system technology to communicate the
+test execution and the HTTP PhoneHome server who receives the PhoneHome request from
```
Comment (flopezag) [inline fiware-region-sanity-tests/README.rst]: "...PhoneHome server who receives..."

who? is a person?

File: fiware-region-sanity-tests/README.rst [hunk 4]
```diff
@@ -225,6 +236,74 @@ provided as part of this component, which has to be installed together with the
...
+The HTTP PhoneHome server waits for *POST HTTP requests* from VMs.
+This service publishes a DBus object (DBus server) to be used by tests to wait for
+PhoneHome requests.
+
+When any request is received, HTTP PhoneHome server will emit over the published object a
+signal to inform to all connected tests about the event (broadcasting). This signal contains the
+hostname of the VM (the one received in the HTTP POST body). This signal will be take in account by
+tests that waits for a signal with this hostname value; rest of tests will ignore it and they keep on
+listening new signals with the correct data to them.
+
```
Comment (flopezag) [inline fiware-region-sanity-tests/README.rst:274]: When a request is received, HTTP PhoneHome server will response, through the published object, a
+signal to inform all connected tests about the event (broadcasting). This signal contains the
+hostname of the VM (the one received in the HTTP POST body). This signal will be take into account by
+tests that are waiting for a signal with this hostname value; the other tests will ignore it and will keep on
+listening new signals with the correct data (i.e. correct hostname) to them.

File: fiware-region-sanity-tests/README.rst [hunk 5]
```diff
@@ -225,6 +236,74 @@ provided as part of this component, which has to be installed together with the
...
+
+**DBus configuration**
+
+The implemented Dbus service uses the *System Bus* for communicating process.
```
Comment (flopezag) [inline fiware-region-sanity-tests/README.rst]: The implemented Dbus service uses the _System Bus_ for communicating processes.
+The bus name used by tests is _org.fiware.fihealth_.
+Additional configuration is needed in `/etc/dbus-1/system.conf` to setup
+the access policies:

**Comment by flopezag** [comment]
LGTM

File: fiware-region-sanity-tests/README.rst [hunk 6]
```diff
@@ -23,6 +23,16 @@ Test environment
 __ `Python - Downloads`_
 __ `Vagrant - Downloads`_
 
+The E2E SNAT test uses DBus tech for communicating process. This one requires:
```
Comment (pratid) [inline fiware-region-sanity-tests/README.rst]: I see no need for details... as E2E tests are part of the test suites, the following 3 items are just more prerequisites.

File: fiware-region-sanity-tests/README.rst [hunk 7]
```diff
@@ -23,6 +23,16 @@ Test environment
...
+
+- `Dbus`_ running and configured on your system
```
Comment (pratid) [inline fiware-region-sanity-tests/README.rst]: Better spelt as "D-Bus"

**Comment by pratid** [comment]
LGTM

**Pull request merged**